predict.py

```diff
@@ -92,7 +92,7 @@ def apply_NMT_model(params, args):
         # True when we should score against a reference
 
         #add the test split reference to the dataset
-        path_list = params['DATA_ROOT_PATH'] + s  + '.' + params['PRED_SCORE']
+        path_list = os.path.join(params['DATA_ROOT_PATH'], s + '.' + params['PRED_SCORE'])
```

Comment:
I suppose we could consider using `pathlib`: https://stackoverflow.com/a/44189428/242457